package.json

```diff
@@ -55,5 +55,6 @@
     "truffle": "^4.1.13",
     "truffle-hdwallet-provider": "0.0.5",
     "web3-utils": "^1.0.0-beta.34"
-  }
+  },
+  "dependencies": {}
```

Comment:
I take it this and commiting the `package-lock.json` isn't intentional?

Reply:
I meant to remove the `ethereumjs-utils` package that we don't need anymore, next commit will do that correctly